src/content/en/tools/chrome-devtools/network-performance/understanding-resource-timing.md

```diff
@@ -2,13 +2,18 @@ project_path: /web/_project.yaml
 book_path: /web/tools/_book.yaml
 description: It is crucial to understand the phases in which resources are gathered over the network. This is the foundation for fixing load issues.
 
-{# wf_published_on: 2016-02-03 #}
+{# wf_published_on: 2016-12-29 #}
 {# wf_updated_on: 2016-02-03 #}
 
 # Understanding Resource Timing {: .page-title }
```

Comment:
See the note in [Understanding Resource Timing](https://github.com/google/WebFundamentals/pull/4000/files#diff-b05a1a6b90e5fd87f39d5e61eb51137a).